src/Game/Game.elm

```diff
@@ -232,7 +232,7 @@ viewStats : Model -> Html Msg
 viewStats ({ hero } as model) =
     div []
         [ div [] [ text "Stats:" ]
-        , div [] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
+        , div [ class (Stats.getHPClass hero.stats) ] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
```

Comment:
I've just looked at Hero and Stats again. The model there is unnecessarily complex, i.e the BaseHero, Stats and Position should probably all be part of the same object along with the hero's attributes and gender. This whole sub-system will require a rewrite. I would advise leaving it for now.

Ultimately, I'd like the Hero to manage it's own rendered output so you can just say `Hero.viewHP model.hero` and put that div somewhere. But you can't do that now.